core/src/main/java/tc/oc/pgm/variables/types/PlayerLocationVariable.java

```diff
@@ -1,19 +1,36 @@
 package tc.oc.pgm.variables.types;
 
 import static java.lang.Math.toRadians;
+import static tc.oc.pgm.util.nms.PlayerUtils.PLAYER_UTILS;
 
+import java.util.function.ToDoubleFunction;
+import org.bukkit.Location;
 import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.util.block.RayBlockIntersection;
 import tc.oc.pgm.variables.VariableDefinition;
 
 public class PlayerLocationVariable extends AbstractVariable<MatchPlayer> {
 
+  private static final double NULL_VALUE = -1000000; // negative 1 million
```

Comment:
Honestly this sounds problematic, if someone tries to start placing blocks 1 million blocks out it's probs gonna force those chunks to load.

just make -1 the default or smthg